pgdrive/envs/multi_agent_pgdrive.py

```diff
@@ -259,7 +259,8 @@ def _after_lazy_init(self):
             top_down_camera_height = self.config["top_down_camera_initial_z"]
             self.main_camera.camera.setPos(0, 0, top_down_camera_height)
             self.main_camera.top_down_camera_height = top_down_camera_height
-            self.main_camera.stop_chase(self.pg_world)
+            self.main_camera.stop_track(self.pg_world, self.current_track_vehicle)
+            self.current_track_vehicle.remove_display_region()
```

Comment:
can we build a function env.stop_track() which call these two functions?

Reply:
It is  redundant now, and I will remove it since remove_display_region() will also be did in stop_track